### PR TITLE
Add event to signal that telescope started/stopped tracking. 

### PR DESCRIPTION
**Pull request by tribeiro**
Updated FakeTelescope to use the new signaling events. Also, on FakeTelescope, added a ``startTracking`` after a ``slewToRaDec`` operation. 

All other external telescope drivers should be updated to include this.

This is actually a major requirement for various reasons, including safety. For instance, rigth now chimera is not aware in case the telescope reaches a low altitute limit as well as if there is any other problem which stops tracking.

### CODE REVIEW

File: src/chimera/interfaces/telescope.py [hunk 1]
```diff
@@ -448,3 +448,26 @@ def isTracking(self):
         @rtype: bool
 
         """
+    @event
+    def trackingStarted(self,position):
```
Comment (phsilva) [inline src/chimera/interfaces/telescope.py]: just add a space between self and position... PEP8 issues...

File: src/chimera/interfaces/telescope.py [hunk 2]
```diff
@@ -448,3 +448,26 @@ def isTracking(self):
...
+        """
+        Indicates that a tracking operation started.
+
+        @param position: The position where the telescope started track.
+        @type  position: L{Position}
+        """
+
+    @event
+    def trackingStopped(self,position, status):
```
Comment (phsilva) [inline src/chimera/interfaces/telescope.py]: just add a space between self and position... PEP8 issues...

**Comment by phsilva** [comment]
Looks nice to me, don't know why we didn't add this before, nice catch.

**Comment by tribeiro** [comment]
Done and done! :smile:

**Comment by ankanaan** [comment]
dumb question:

As all external drivers need this feature, it would be nice to make it
happen somehow somewhere :-)  Or at least to force these drivers to
implement it.  Do you see what I mean?  Is this possible?  As a good drunk
native English speaker would say: "I mean, you know..."

**Pull request merged**